src/editors/containers/TextEditor/components/ImageSettingsModal/index.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Form,
+  Image,
+} from '@edx/paragon';
+
+import BaseModal from '../BaseModal';
+import * as module from '.';
+
+export const hooks = {
```

Comment:
I forget, are we separating these hooks out into there own file or have I converted you?